contracts/GEONToken.sol

```diff
@@ -17,6 +17,8 @@ import 'openzeppelin-solidity/contracts/ownership/CanReclaimToken.sol';
 import './ERC223ReceivingContract.sol';
 
 contract GEONToken is StandardToken, RBACMintableToken, PausableToken, BurnableToken, CanReclaimToken {
+	uint256 public constant MAX_SUPPLY = 850000000 * 1e18;
```

Comment:
We're using number precision which should be the same as token's `decimals`, so it's safer to reuse it here.

```suggestion
	uint256 public constant MAX_SUPPLY = 850000000 * (10 ** uint256(decimals));
```